gravitee-apim-portal-webui-next/projects/gravitee-dashboard/src/lib/components/chart/pie-chart/pie-chart.component.stories.ts

```diff
@@ -17,16 +17,15 @@ import { applicationConfig, Meta, moduleMetadata, StoryObj } from '@storybook/an
 import { provideCharts, withDefaultRegisterables } from 'ng2-charts';
 
 import { PieChartComponent, PieType } from './pie-chart.component';
+import { MetricsResponse } from '../../widget/widget';
 
 interface PieChartStoryArgs {
   storyId?: string;
   type: PieType;
-  data: {
-    labels: string[];
-    datasets: Array<{
-      data: number[];
-    }>;
-  };
+  buckets: Array<{
+    key: string;
+    value: number;
+  }>;
```

Comment:
Feels a little java-y 🤔 
```suggestion
  buckets: {
    key: string;
    value: number;
  }[];
```